### DISCUSSION TITLE
Feature request: Run the test command in the tmux panel

### DISCUSSION THREAD

**Discussion by damnever** [nvim-neotest/neotest#473] [Ideas]
[vim-test](https://github.com/vim-test/vim-test?tab=readme-ov-file#strategies) provides various strategies to run test commands in a separate tmux pane. It is useful when we want to tweak the test command for further investigation, for example, running `go test` with profile arguments.

**Comment by jweslley** [comment]
I created a custom tmux strategy for myself, and I have been using it for the last 2 weeks without major surprises. It has hard-coded tmux settings based on personal preferences but could be tweaked. 

https://github.com/jweslley/dotfiles/commit/4906bbea7d8d787dcede516bbb336c14279fe7b6
